docs/04-reference/wingsdk-api.md

```diff
@@ -2084,14 +2083,6 @@ to search for.
 
 ---
 
-##### `copy` <a name="copy" id="@winglang/sdk.std.ImmutableArray.copy"></a>
```

Comment:
hmm, I dont think there are any changes in this PR that would have removed these. Is it an artifact from local development that we dont want to merge?

Reply:
It's changes I forgot to add to my recent PR :) I guess Tsuf just ran `npm run build` on the wingsdk folder